valet-key/Readme.md

```diff
@@ -4,47 +4,61 @@ This document describes the Valet Key Pattern example from the guide [Cloud Desi
 
 ## System Requirements
 
-* Microsoft .NET Framework version 4.5
-* Microsoft Visual Studio 2015 Community, Enterprise, or Professional
-* Windows Azure SDK for .NET version 2.9
+* Microsoft .NET Framework 5
+* Microsoft Visual Studio 2019 or Later
+* Windows Azure SDK for .NET release 2021
 
 ## Before you start
 
 Ensure that you have installed all of the software prerequisites.
-
-The example demonstrates operational aspects of applications running in Windows Azure. Therefore, you will need to use the diagnostics tools in order to understand how the code sample works. You **must** ensure that the web and worker roles in the solution are configured to use the diagnostics mechanism. If not, you will not see the trace information generated by the example.
+If you don't have one, provision an Azure Storage account from the Azure Portal.
+Once you storage account is created, add a container to it, name it "valetkeysample".
 
 ## About the Example
  
 This example shows how a client application can obtain a shared access signature with the necessary permissions to write directly to blob storage. For simplicity, this sample focuses on the mechanism to obtain and consume a valet key and does not show how to implement authentication or secure communications.
 
-
 ## Running the Example
 
-You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure Cloud Service.
+You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure App Service.
```

Comment:
> Visual Studio Windows Azure emulator.

Do you mean the Microsoft Azure Compute Emulator? I don't think we have called it "Windows Azure" for a long time.  And isn't that already installed with SDK?  Might not be important to call out specifically vs just saying that you can run this locally or deploy it to Azure.  (which again you have it still called "Windows Azure")

Reply:
Done